src/CMakeLists.txt

```diff
@@ -3,19 +3,20 @@ add_library(UDBM STATIC DBMAllocator.cpp dbm.c fed_dbm.cpp mingraph.c mingraph_r
         priced.cpp valuation.cpp)
 set_property(TARGET UDBM PROPERTY C_VISIBILITY_PRESET hidden)
 set_property(TARGET UDBM PROPERTY VISIBILITY_INLINES_HIDDEN ON)
-set_target_properties(UDBM PROPERTIES CXX_CLANG_TIDY "${CLANG_TIDY_COMMAND}")
-target_compile_definitions(UDBM INTERFACE XXHASH_INLINE_ALL)
+if (NOT CMAKE_SYSTEM_NAME STREQUAL Windows) # unknown argument: '-fno-keep-inline-dllexport'
+    set_target_properties(UDBM PROPERTIES CXX_CLANG_TIDY "${CLANG_TIDY_COMMAND}")
+endif()
 target_link_libraries(UDBM PRIVATE UUtils::base UUtils::udebug UUtils::hash)
 
 target_include_directories(UDBM
     PRIVATE
         # where the library itself will look for its internal headers
-        ${CMAKE_CURRENT_BINARY_DIR}/../include
-        ${CMAKE_CURRENT_SOURCE_DIR}/../include
+        ${PROJECT_BINARY_DIR}/include
+        ${PROJECT_SOURCE_DIR}/include
     PUBLIC
         # where top-level project will look for the library's public headers
-        $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/../include>
-        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>
+        $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>
```

Comment:
It doesn't really make sense to add the `.._BINARY_DIR` to include directories. Try removing 
`${PROJECT_BINARY_DIR}/include`
and
`$<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>`

Reply:
Nice catch!